css/grid.css

```diff
@@ -0,0 +1,445 @@
+}
+
+.button:hover {
+	background: #7e7e7e;
+	border-color: #7e7e7e;
+}
+
+.button-blue {
+	background: #6abfc3;
+	border-color: #6abfc3;
+}
+
+.button-blue:hover {
+	background: #87cbce;
+	border-color: #87cbce;
+}
+
+.button-yellow {
+	border-color: #cc9543;
+	background: #cc9543;
+}
+
+.button-yellow:hover {
+	background: #d4a866;
+	border-color: #d4a866;
+}
+
+/* #endregion */
+
+/* #region spacing */
+
+.spacing-1 {
+	margin-top: 20px;
+	margin-bottom: 20px;
+}
+
+.spacing-2 {
+	margin-top: 40px;
+	margin-bottom: 40px;
+}
+
+.spacing-3 {
+	margin-top: 60px;
+	margin-bottom: 60px;
+}
+
+.spacing-4 {
+	margin-top: 80px;
+	margin-bottom: 80px;
+}
+
+.spacing-5 {
+	margin-top: 100px;
+	margin-bottom: 100px;
+}
+
+/* #endregion */
+
+/* #region navbar */
+
+.navbar {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.nav-row {
+	flex-direction: row;
+}
+
+.nav-col {
+	flex-direction: column;
+}
+
+.nav-item {
+	text-align: center;
+	margin-left: 10px;
+	margin-right: 10px;
+}
+
+.nav-menu {
+	display: none;
+}
+
+/* #endregion */
+
+/* #region grids */
+
+.grid-start-1 {
+	grid-column-start: 1;
+}
+
+.grid-start-2 {
+	grid-column-start: 2;
+}
+
+.grid-start-3 {
+	grid-column-start: 3;
+}
+
+.grid-start-4 {
+	grid-column-start: 4;
+}
+
+.grid-start-5 {
+	grid-column-start: 5;
+}
+
+.grid-start-6 {
+	grid-column-start: 6;
+}
+
+.grid-start-7 {
+	grid-column-start: 7;
+}
+
+.grid-start-8 {
+	grid-column-start: 8;
+}
+
+.grid-start-9 {
+	grid-column-start: 9;
+}
+
+.grid-start-10 {
+	grid-column-start: 10;
+}
+
+.grid-start-11 {
+	grid-column-start: 11;
+}
+
+.grid-start-12 {
+	grid-column-start: 12;
+}
+
+.grid-1,
+.grid-md-1,
+.grid-sm-1 {
+	grid-column-end: span 1;
+}
+
+.grid-2,
+.grid-md-2,
+.grid-sm-2 {
+	grid-column-end: span 2;
+}
+
+.grid-3,
+.grid-md-3,
+.grid-sm-3 {
+	grid-column-end: span 3;
+}
+
+.grid-4,
+.grid-md-4,
+.grid-sm-4 {
+	grid-column-end: span 4;
+}
+
+.grid-5,
+.grid-md-5,
+.grid-sm-5 {
+	grid-column-end: span 5;
+}
+
+.grid-6,
+.grid-md-6,
+.grid-sm-6 {
+	grid-column-end: span 6;
+}
+
+.grid-7,
+.grid-md-7,
+.grid-sm-7 {
+	grid-column-end: span 7;
+}
+
+.grid-8,
+.grid-md-8,
+.grid-sm-8 {
+	grid-column-end: span 8;
+}
+
+.grid-9,
+.grid-md-9,
+.grid-sm-9 {
+	grid-column-end: span 9;
+}
+
+.grid-10,
+.grid-md-10,
+.grid-sm-10 {
+	grid-column-end: span 10;
+}
+
+.grid-11,
+.grid-md-11,
+.grid-sm-11 {
+	grid-column-end: span 11;
+}
+
+.grid-12,
+.grid-md-12,
+.grid-sm-12 {
+	grid-column: span 12;
+}
+
+@media only screen and (min-width: 721px) and (max-width: 1024px) {
+	.grid-md-1,
+	.grid-md-2,
+	.grid-md-3,
+	.grid-md-4,
+	.grid-md-5,
+	.grid-md-6,
+	.grid-md-7,
+	.grid-md-8,
+	.grid-md-9,
+	.grid-md-10,
+	.grid-md-11,
+	.grid-md-12 {
+		grid-column: span 12;
+	}
+
+	.nav-item {
+		display: none;
+	}
+
+	.nav-menu {
+		display: initial;
+	}
+}
+
+@media only screen and (max-width: 720px) {
+	.grid-md-1,
+	.grid-md-2,
+	.grid-md-3,
+	.grid-md-4,
+	.grid-md-5,
+	.grid-md-6,
+	.grid-md-7,
```

Comment:
`.grid-md` classes should go inside the media queries for medium screen width. :no_entry:

Reply:
These classes are already on the media query for the medium screen, as you can see from the line 384 to the line 392. The classes you're seeing here is to specify such classes will remain taking the whole 12 grids when the screen is small. These lines are necessary to make the classes work as intended, just like in Bootstrap.